x/ibc/04-channel/keeper/timeout.go

```diff
@@ -120,21 +147,31 @@ func (k Keeper) TimeoutOnClose(
 	packet types.Packet,
 	proofNonMembership,
 	proofClosed commitment.ProofI,
-	proofHeight uint64,
-	portCapability sdk.CapabilityKey,
+	proofHeight,
+	nextSequenceRecv uint64,
 ) (exported.PacketI, error) {
 	channel, found := k.GetChannel(ctx, packet.GetSourcePort(), packet.GetSourceChannel())
 	if !found {
 		return nil, sdkerrors.Wrapf(types.ErrChannelNotFound, packet.GetSourcePort(), packet.GetSourceChannel())
 	}
 
-	_, found = k.GetChannelCapability(ctx, packet.GetSourcePort(), packet.GetSourceChannel())
-	if !found {
-		return nil, types.ErrChannelCapabilityNotFound
-	}
+	// TODO: uncomment
```

Comment:
Is this also related to the keys work? Might be helpful to have a common comment here so that all the places in the code can be quickly found and changed.